packages/components/src/components/carousel-item/carousel-item.stories.ts

```diff
@@ -4,8 +4,16 @@ const { argTypes, parameters } = storybookDefaults('sd-carousel-item');
 const { overrideArgs } = storybookHelpers('sd-carousel-item');
 const { generateTemplate } = storybookTemplate('sd-carousel-item');
 
+/**
+ * A carousel item represents a slide within a carousel.
+ *
+ * **Related templates**:
+ * - [Carousel with Images](?path=/docs/templates-carousel-with-images--docs)
```

Comment:
Link is not working

Reply:
In the current branch this is only a placeholder for the template which was created in the sd-carousel docs branch. It will work once both are merged